sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -912,16 +931,16 @@ private void ThrowIfSessionLockLost()
             {
                 amqpRequestMessage.AmqpMessage.ApplicationProperties.Map[ManagementConstants.Request.AssociatedLinkName] = receiveLink.Name;
             }
-            amqpRequestMessage.Map[ManagementConstants.Properties.LockTokens] = new[] { new Guid(lockToken) };
+            amqpRequestMessage.Map[ManagementConstants.Properties.LockTokens] = new List<Guid>(1) { new Guid(lockToken) };
```

Comment:
I tried to do the necessary research in the AMQP lib used but couldn't figure out if the type here really has to be an array. I was hopping the tests would capture it

Reply:
AMQP encoding is different for both the types. Service Bus service expects ArrayEncoding here, if we pass list service would throw an error.

Reply:
Thanks for the investigation but looks like we need to revert this.